ghost/core/core/server/data/migrations/versions/5.93/2024-09-03-20-09-40-null-analytics-jobs-timings.js

```diff
@@ -0,0 +1,15 @@
+// For information on writing migrations, see https://www.notion.so/ghost/Database-migrations-eb5b78c435d741d2b34a582d57c24253
+
+// For DML - data changes
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        await knex.raw(`
+            DELETE FROM jobs
+            WHERE name = 'email-analytics-latest-opened' OR name = 'email-analytics-latest-others' OR name = 'email-analytics-missing';
+        `);
```

Comment:
We may want to wrap this in a try/catch with some error logging, just in case there's some kind of DB error

Reply:
Added.